### DISCUSSION TITLE
Clicked an Imposter Discord App and directed to Phantom Blocklist Page - Am I Safe???

### DISCUSSION THREAD

**Discussion by amerv2** [phantom/docs#360] [Help & Support] [answered]
### Summary

Hello, 

I'm kinda panicking...

I was in a rush and accidentally clicked the link to a fake Collab land discord app. I authorized my Discord (which I've now removed) and was directed to a website that was on the Phantom Blocklist. I never went to the site and have removed the Discord app. Am I okay? I don't think anything was downloaded to my computer locally, but wanted to check here. Did the Phantom Blocklist basically save me or am I still at risk of having my wallets, emails, device compromised?

Thank you!

### Steps to Reproduce

I went to the discord link in the Tessara Twitter profile (https://x.com/tessera). Within the discord I clicked the fake collab link and authorized my discord which then took me to a webpage that Phantom fortunately blocked via the Phantom Blocklist. 

### Phantom Version

24.30.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hello @amerv2,

For this issue please reach out to user support via settings in your phantom wallet.
